.prettierrc

```diff
@@ -1,6 +1,7 @@
 {
   "tabWidth": 2,
   "useTabs": false,
+  "singleQuote": true,
```

Comment:
Can you remove the single quote and keep everything as double quotes to avoid so many changes?